### PR TITLE
Update trackers and add ptpimg uploads

### PR DESCRIPTION
**Pull request by 8b00001000**
**Currently functional but needs new documentation!**

- [x] Remove STP & WCD
- [x] Update Bib url
- [x] Add PTH
- [x] Add ptpimg
- [x] Python 3 fixes
- [x] https://github.com/pthcode/alexandria/issues/2 Grab full description
- [x] Make console output prettier
- [ ] Update documentation

### CODE REVIEW

File: alexandria [hunk 1]
```diff
@@ -219,60 +219,13 @@ class WhatCD(Tracker):
             'image': metadata['image'],
             'desc': metadata['description'],
         }
-
-        r = self.session.post('https://what.cd/upload.php', files=files, data=data)
+        r = self.session.post('https://passtheheadphones.me/upload.php', files=files, data=data)
         return True
 
     def has_book(self, metadata):
+        print
```
Comment (zacharydenton) [inline alexandria]: Please remove.

File: alexandria [hunk 2]
```diff
@@ -344,19 +297,21 @@ def determine_format(filename):
 def create_tempfile(suffix):
     return subprocess.check_output(['mktemp', '-u', '--suffix', suffix]).strip()
 
-def make_torrent(filename, announce, piece_size=16):
+def make_torrent(filename, announce, tracker_name, piece_size=16):
```
Comment (zacharydenton) [inline alexandria:299]: Hmm. I should update libpth.utils.make_torrent to work for multiple trackers.

**Review by zacharydenton — APPROVED** [review]
Looks great. Thanks for bringing this tool back from the dead!

**Comment by 8b00001000** [comment]
Will hopefully address issues and finish this up today.

**Review by zacharydenton — APPROVED** [review]

**Pull request merged**